src/resolve-protocol.ts

```diff
@@ -12,7 +12,8 @@ const connect = async (proxyUrl: string, options: tls.ConnectionOptions, callbac
         host = `[${options.host}]:${options.port}`;
     }
 
-    (async () => {
+    // eslint-disable-next-line no-void
```

Comment:
we really need to get rid of the `no-void` rule as well

Reply:
I've mentioned https://github.com/iCrawl/eslint-config-neon before, our eslint rules really need some love and attention 😢